### DISCUSSION TITLE
Question 55: How best do we deal with the issue of bottom depth with regards to georeferencing marine sites? [IZ - KJ, RDLP, JD]

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#829] [Q&A] [answered]
Description: Georeferencers working on IZ projects use dredging depth to determine unique coordinates and error radii based on bathymetric maps. Depth is recorded in collection events, not sites. A single site may be attached to multiple collection events, each CE with a different dredge depth. There is a need to assign/record different coordinates in InvertEBase and EMu for a single site based on dredge depth, so each dredge depth should be represented by a distinct site record. We have come up with the following ways to create sites for unique dredge depths - which is best? 

1. Adding depth to the Sites module on the Mapping tab opposite Elevation (Above Sea Level) with [funding for dev].  Maybe where ‘Elevation (Relative)’ currently resides?(Is this possible? How much \$\$?).  We found only 7 records across other divisions using this section. It’s unclear the distinction between Relative and ASL, and whether the values in these 7 records are even valid as “Relative” (whatever that means)?
2. Use the elevation field in the Sites module and record negative elevation for depth.  (not very orthodox, admittedly)
3. Create a unique site for each instance of unique dredge depth, and connect these sites to collecting events containing the dredge depths via a matching site identifier and collection event code. 
4. Create a unique site for each instance of unique dredge depth by recording dredge depth in precise locality.
5. Create a unique site for each instance of unique dredge depth and recording dredge depth in the Remarks or Notes field.

**Comment by rondlg** [comment]
Response: Transferred to esites working group discussion on 9th May 2023

- Agenda is here: https://github.com/fieldmuseum/EMu-Documentation/discussions/811

**Comment by rondlg** [comment, marked as answer]
**Immediate solution:** Use Elevation (Relative) for depth below surface. The values here will be negative. Corresponds to [DistanceAboveSurfaceInMeters](https://dwc.tdwg.org/list/#dwc_maximumDistanceAboveSurfaceInMeters). Update Field Level Help.

- Should bottom depth in CE also go in Elevation (relative) in Sites for other areas?
- - **Yes**. While depth may change; in Sites, the depth/elevation-data is part of the geolocation of a given locality [at a given time]

- Should we use meters or feet ?
- - **Feet**. In this interim phase the feet fields calculation is more accurate to fathoms than the meters fields.


Over time, can relate overlapping older/new sites to each other view synonymy-workflows (e.g. [here](https://docs.google.com/document/d/1YA87zYvahTko2McaCWmTofg7XBdjKAbfZjJSG6KfS6Q/edit#heading=h.aux68lr9iqu1))

**Reply by fmrdelapena** [reply]
Question -- "The values here will be negative" -- does this mean we should enter them as negative or they will be converted to negative?  Nothing in this table currently indicates these values represent depth below surface.

**Comment by rondlg** [comment]
EMu will not convert to negatives for you so you should enter them as a negative.

![image](https://github.com/fieldmuseum/EMu-Documentation/assets/702709/16cd6d0a-b304-4487-9f5e-90d8dd1b621b)

We have also updated the field level help for the Relative Elevation fields.

**Reply by fmrdelapena** [reply]
Are we supposed to be manually copying the depth values from the Collection Events into the Relative Table or will IT be migrating that data for us.  We've been adding it when we create new site records, but if wasn't clear how the existing records with depth values will be handled.

**Comment by rondlg** [comment]
See also: https://github.com/fieldmuseum/EMu-Documentation/issues/12
